README.md

```diff
@@ -59,6 +59,100 @@ Use the ``Deploy to Bluemix`` button **OR** create the services and run locally.
     * wbc-natural-language-understanding-service
     * wbc-tone-analyzer-service
 
+## Deploy to Kubernetes on Bluemix
+
+The following steps outline the process for deploying this application as-is to Kubernetes on IBM Bluemix.  You can read more about [Kubernetes Support and the IBM Cloud Developer Tools CLI on the Bluemix blog](https://www.ibm.com/blogs/bluemix/2017/09/deploying-kubernetes-ibm-cloud-ibm-cloud-developer-tools-cli/).
+
+1. [Create a Kubernetes cluster on Bluemix](https://console.bluemix.net/containers-kubernetes/launch?) if you do not already have one. 
+
+   *Note: This may take up to 15 minutes for a cluster to be provisioned.*
+
+2. Install the [IBM Cloud Developer Tools CLI](https://github.com/IBM-Bluemix/ibm-cloud-developer-tools/)
+
+    If you're on a MacOS or Linux machine, open up a terminal and execute the following command:
+
+    ```
+    curl -sL https://ibm.biz/idt-installer | bash
+    ```
+
+    If you're on a windows machine, open Windows PowerShell by right-clicking and select "Run as Administrator":
+
+    ```
+    Set-ExecutionPolicy Unrestricted; iex(New-Object Net.WebClient).DownloadString('http://ibm.biz/idt-win-installer')
+    ``` 
+
+    This will install the IBM Cloud Developer Tools CLI, the Bluemix CLI, Docker, the Kubernetes CLI, and all other dependencies if they are not already installed.
```

Comment:
I don't think "and all other dependencies" is particularly in the spirit of journeys. Ideally, the developer should be able to skip a script that does all the work in favor of doing as much of it as possible using documentation alone. In this case, I had to go read exactly what https://ibm.biz/idt-installer actually installs in otinstalls in order to follow along.